### PR TITLE
UX: Add descriptions to auto groups and auto group indicator

### PR DESCRIPTION
**Pull request by martin-brennan**
This commit adds a description for all the auto groups
which will be shown in the group list and show group
pages, which will help admins understand their purpose
better.

Also adds an indicator with a tooltip to explain what
the auto groups are on the group show page.

image
image

### CODE REVIEW

**Comment by SamSaffron** [comment]
Just wanted to say :hugs: I think this is a significant help for new users on the platform!

**Comment by martin-brennan** [comment]
Thanks @SamSaffron I appreciate it :hugs:

**Review by ZogStriP — APPROVED** [review]
👏

**Comment by martin-brennan** [comment]
@moin-Jana good point, I will alter the descriptions to make it clear the members of TL groups are also members of all lower TL groups

File: app/serializers/basic_group_serializer.rb [hunk 1]
```diff
@@ -44,10 +44,16 @@ def display_name
...
+  def bio_cooked
+    if object.automatic
+      return I18n.t("groups.default_descriptions.#{Group::AUTO_GROUP_IDS[object.id]}")
     end
+
+    object.bio_cooked
+  end
+
+  def bio_excerpt
+    PrettyText.excerpt(bio_cooked, 110, keep_emoji_images: true) if bio_cooked.present?
```
[the remark this answers is not in the document]
Comment (martin-brennan) [reply app/serializers/basic_group_serializer.rb]: Bumped it to 200 to accommodate some of the new descriptions

**Pull request merged**